packages/splits-sdk-react/src/hooks/splitV1.ts

```diff
@@ -29,10 +24,8 @@ export const useSplitsClient = (config?: SplitsClientConfig): SplitsClient => {
     throw new Error('Make sure to include <SplitsProvider>')
   }
 
-  const chainId =
-    config && 'chainId' in config
-      ? config.chainId
-      : context.splitsClient._chainId
+  const apiConfig = config && config.apiConfig
+  const chainId = config && (config.chainId as number)
```

Comment:
there was a specific reason i set things up the way i did there. can see the notes in https://github.com/0xSplits/splits-sdk/pull/27

Should have left a comment explaining that.

Unless you made this change on purpose? Happy to chat if so. Or can do async here.

Reply:
okay, reverting this, thought it was redundant when I looked at it and hence removed it